torchaudio/functional.py

```diff
@@ -254,59 +244,93 @@ def istft(stft_matrix,          # type: Tensor
 
 
 @torch.jit.script
-def spectrogram(sig, pad, window, n_fft, hop, ws, power, normalize):
+def spectrogram(waveform, pad, window, n_fft, hop_length, win_length, power, normalized):
     # type: (Tensor, int, Tensor, int, int, int, int, bool) -> Tensor
     r"""Create a spectrogram from a raw audio signal.
 
     Args:
-        sig (torch.Tensor): Tensor of audio of size (c, n)
+        waveform (torch.Tensor): Tensor of audio of size (c, n)
         pad (int): Two sided padding of signal
-        window (torch.Tensor): Window_tensor
+        window (torch.Tensor): Window tensor that is applied/multiplied to each frame/window
         n_fft (int): Size of fft
-        hop (int): Length of hop between STFT windows
-        ws (int): Window size
-        power (int) : Exponent for the magnitude spectrogram,
+        hop_length (int): Length of hop between STFT windows
+        win_length (int): Window size
+        power (int): Exponent for the magnitude spectrogram,
             (must be > 0) e.g., 1 for energy, 2 for power, etc.
-        normalize (bool) : Whether to normalize by magnitude after stft
+        normalized (bool): Whether to normalize by magnitude after stft
 
     Returns:
-        torch.Tensor: Channels x hops x n_fft (c, l, f), where channels
-        is unchanged, hops is the number of hops, and n_fft is the
-        number of fourier bins, which should be the window size divided
-        by 2 plus 1.
+        torch.Tensor: Channels x frequency x time (c, f, t), where channels
+        is unchanged, frequency is `n_fft // 2 + 1` where `n_fft` is the number of
+        fourier bins, and time is the number of window hops (n_frames).
     """
-    assert sig.dim() == 2
+    assert waveform.dim() == 2
 
     if pad > 0:
         # TODO add "with torch.no_grad():" back when JIT supports it
-        sig = torch.nn.functional.pad(sig, (pad, pad), "constant")
+        waveform = torch.nn.functional.pad(waveform, (pad, pad), "constant")
 
     # default values are consistent with librosa.core.spectrum._spectrogram
-    spec_f = _stft(sig, n_fft, hop, ws, window,
-                   True, 'reflect', False, True).transpose(1, 2)
+    spec_f = _stft(waveform, n_fft, hop_length, win_length, window,
+                   True, 'reflect', False, True)
 
-    if normalize:
+    if normalized:
         spec_f /= window.pow(2).sum().sqrt()
-    spec_f = spec_f.pow(power).sum(-1)  # get power of "complex" tensor (c, l, n_fft)
+    spec_f = spec_f.pow(power).sum(-1)  # get power of "complex" tensor
     return spec_f
 
 
 @torch.jit.script
-def create_fb_matrix(n_stft, f_min, f_max, n_mels):
+def spectrogram_to_DB(specgram, multiplier, amin, db_multiplier, top_db=None):
```

Comment:
Did you just move this around? Why?

Reply:
https://github.com/pytorch/audio/blob/master/torchaudio/functional.py#L11
It makes sense to have `spectrogram` and `spectrogram_to_DB` beside each other and `create_fb_matrix` and `create_dct` beside each other.

Reply:
ok, then lets do that in a follow-up PR, but not while changing them at the same time